x-pack/metricbeat/module/gcp/compute/_meta/docs.asciidoc

```diff
@@ -1,96 +1,30 @@
 Compute metricset to fetch metrics from https://cloud.google.com/compute/[Compute Engine] Virtual Machines in Google Cloud Platform. No Monitoring or Logging agent is required in your instances to use this metricset.
 
-The `compute` metricset contains all metrics exported from the https://cloud.google.com/monitoring/api/metrics_gcp#gcp-compute[Stackdriver API]. The field names have been left untouched for people already familiar with them.
+The `compute` metricset contains all metrics exported from the https://cloud.google.com/monitoring/api/metrics_gcp#gcp-compute[Stackdriver API]. The field names are aligned to https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html[Beats naming conventions] with minor modifications to their GCP metrics name counterpart.
 
 Extra labels and metadata are also extracted using the https://cloud.google.com/compute/docs/reference/rest/v1/instances/get[Compute API]. This is enough to get most of the info associated with a metric like Compute labels and metadata and metric specific Labels.
 
 [float]
-=== Metrics and labels
-Here is a list of metrics collected by `compute` metricset:
+=== Labels
+Here is a list of labels collected by `compute` metricset depending on the type of metric being collected:
 
-[float]
-==== firewall
-
-* `instance.firewall.dropped_bytes_count`: Incoming bytes dropped by the firewall.
-- `instance_name`: The name of the VM instance.
-
-* `instance.firewall.dropped_packets_count`: Incoming packets dropped by the firewall.
-- `instance_name`: The name of the VM instance.
-
-[float]
-==== cpu
-
-* `instance.cpu.reserved_cores`: Number of cores reserved on the host of the `instance`.
-- `instance_name`: The name of the VM instance.
-
-* `instance.cpu.utilization`: The fraction of the allocated CPU that is currently in use on the `instance`.
-- `instance_name`: The name of the VM instance.
-
-* `instance.cpu.usage_time`: Usage for all cores in seconds.
-- `instance_name`: The name of the VM instance.
-
-[float]
-==== disk
-
-* `instance.disk.read_bytes_count`: Count of bytes read from disk.
-- `instance_name`: The name of the VM instance.
-- `device_name`: The name of the disk device.
-- `storage_type`: The storage type: `pd-standard`, `pd-ssd`, or `local-ssd`.
-- `device_type`: The disk type: `ephemeral` or `permanent`.
-
-* `instance.disk.read_ops_count`: Count of disk read IO operations.
-- `instance_name`: The name of the VM instance.
-- `device_name`: The name of the disk device.
-- `storage_type`: The storage type: `pd-standard`, `pd-ssd`, or `local-ssd`.
-- `device_type`: The disk type: `ephemeral` or `permanent`.
-
-* `instance.disk.write_bytes_count`: Count of bytes written to disk
 - `instance_name`: The name of the VM instance.
+    Collected with:
```

Comment:
I don't think the layout here is what you expect: https://beats_33006.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/metricbeat-metricset-gcp-compute.html

You probably need a continuation character and some line breaks. Check out our internal docs about list formatting [here](https://github.com/elastic/docs/blob/master/README.asciidoc#lists).

Reply:
Thanks, I fixed it in [`7bfe982` (#33006)](https://github.com/elastic/beats/pull/33006/commits/7bfe982f92c47e53e983c975ca4429a595eee40d)